src/tomato/main.py

```diff
@@ -37,6 +37,13 @@ def _default_parsers() -> tuple[argparse.ArgumentParser, argparse.ArgumentParser
         default=False,
         help="Launch tomato in test mode.",
     )
+    parser.add_argument(
+        "-J",
+        "--json",
+        action="store_true",
+        default=False,
+        help="Return a json string from ketchup commands.",
+    )
```

Comment:
Rather than adding this here, it would be perhaps more correct to add it just to `status` and `submit`? Otherwise the argument shows up in the help for all subcommands, I think.